src/backend/launcher.ts

```diff
@@ -769,20 +770,51 @@ interface RunnerProps {
 
 const commandsRunning = {}
 
+let shouldUsePowerShell: boolean | null = null
+
 async function callRunner(
   commandParts: string[],
   runner: RunnerProps,
   abortController: AbortController,
   options?: CallRunnerOptions,
   gameInfo?: GameInfo
 ): Promise<ExecResult> {
-  const fullRunnerPath = join(runner.dir, runner.bin)
   const appName = commandParts[commandParts.findIndex(() => 'launch') + 1]
 
   // Necessary to get rid of possible undefined or null entries, else
   // TypeError is triggered
   commandParts = commandParts.filter(Boolean)
 
+  let bin = runner.bin
+  let fullRunnerPath = join(runner.dir, bin)
+
+  // macOS/Linux: `spawn`ing an executable in the current working directory
+  // requires a "./"
+  if (!isWindows) bin = './' + bin
+
+  // On Windows: Use PowerShell's `Start-Process` to wait for the process and
+  // its children to exit, provided PowerShell is available
+  if (shouldUsePowerShell === null) {
+    shouldUsePowerShell =
+      isWindows && !!(await searchForExecutableOnPath('powershell'))
```

Comment:
might be more readable if we just assign this to a variable first 
`const powershellPath = await searchForExecutableOnPath('powershell')`